src/QRCoder/manifest.json

```
@@ -0,0 +1,11 @@
+{
```

内嵌依赖之后要将`Dependencies`设为空。否则会被APM装依赖
```suggestion
{
"Dependencies": [],
```